### PR TITLE
remove extra period

### PR DESCRIPTION
**Pull request by mairaw**
Extra period on the note is weird:
image

### CODE REVIEW

File: docs/core/windows-prerequisites.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ See [.NET Core 1.x Supported OS Versions](https://github.com/dotnet/core/blob/ma
 
 > [!NOTE]
 > *For Windows 8.1 and earlier versions, or Windows Server 2012 R2 and earlier versions:*
-> Make sure that your Windows installation is up-to-date and includes [KB2999226](https://support.microsoft.com/en-us/help/2999226/update-for-universal-c-runtime-in-windows) which can be installed through Windows Update. If you don't have this update installed, you'll see an error when you launch a .NET Core application like the following: `The program can't start because api-ms-win-crt-runtime-1-1-0.dll is missing from your computer. Try reinstalling the program to fix this problem.`.
+> Make sure that your Windows installation is up-to-date and includes [KB2999226](https://support.microsoft.com/en-us/help/2999226/update-for-universal-c-runtime-in-windows) which can be installed through Windows Update. If you don't have this update installed, you'll see an error when you launch a .NET Core application like the following: `The program can't start because api-ms-win-crt-runtime-1-1-0.dll is missing from your computer. Try reinstalling the program to fix this problem.`
```
Comment (rpetrusha) [inline docs/core/windows-prerequisites.md]: unrelated to your PR: it would be best to rephrase this as "you'll see an error like the following when you launch a .NET Core application." Here, the string applies to the application, not the error.

**Review by rpetrusha — APPROVED** [review]
Left one comment unrelated to your PR, #mairaw. You can decide whether you want to address it or not.

**Comment by mairaw** [comment]
I'll have to create a new PR for this topic, so I'll address your feedback with that change. Thanks @rpetrusha!
